custom_components/waste_collection_schedule/waste_collection_schedule/source/plano_gov.py

```diff
@@ -0,0 +1,222 @@
+import requests
+import logging
+import datetime
+import holidays
+from waste_collection_schedule import Collection
+
+TITLE = "City of Plano" # Title will show up in README.md and info.md
+DESCRIPTION = "Source script for plano.gov"  # Describe your source
+COUNTRY = "us"  # ISO 3166-1 alpha-2 country code, e.g. "DE" for Germany, "US" for United States
+URL = "https://www.plano.gov/630/Residential-Collection-Schedules"  # Insert url to service homepage. URL will show up in README.md and info.md
+TEST_CASES = {  # Insert arguments for test cases to be used by test_sources.py script
+    "GoodObjectId": {"arg1": "00000","arg2":"3"},  # Example object ID, replace with a valid one
+    "GoodObjectId": {"arg1": "00000","arg2":"5"},  # Example object ID, replace with a valid one
+}
+
+API_URL = "https://maps.planogis.org/arcgiswad/rest/services/Sustainability/ServicedAddresses/MapServer/0/query?"
+ICON_MAP = {   # Optional: Dict of waste types and suitable mdi icons
+    "TRASH": "mdi:trash-can",
+    "RECYCLE": "mdi:recycle",
+    "BULKY": "mdi:size-xl",
+}
+
+
+#### Arguments affecting the configuration GUI ####
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = { # Optional dictionary to describe how to get the arguments, will be shown in the GUI configuration form above the input fields, does not need to be translated in all languages
+    "en": "HOW TO GET ARGUMENTS DESCRIPTION",
+    "de": "WIE MAN DIE ARGUMENTE ERHÄLT",
+    "it": "COME OTTENERE GLI ARGOMENTI",
+    "fr": "COMMENT OBTENIR LES ARGUMENTS",
+}
```

Comment:
Either fill in or remove this, as now the "HOW TO GET ARGUMENTS DESCRIPTION" will show up in the configuration window. (see changed en.json file)

> Configure your service provider. 
>
>HOW TO GET ARGUMENTS DESCRIPTION
>
>More details: https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/plano_gov.md.